db/seeds.rb

```diff
@@ -323,3 +323,12 @@ def create_repository(link:, company:)
   repositories: repos,
   dates: contributions_dates
 )
+
+print 'Creating mentors to some users..'
+
+30.times do
+  mentor = User.all.sample
+  mentee = User.all.sample.update(reviewer_id: mentor.id)
```

Comment:
What is the need for this assign?

Reply:
You're right. There is not need for this `mentee` assign.